website/content/docs/commands/operator/snapshot/restore.mdx

```diff
@@ -46,4 +46,4 @@ nomad operator snapshot restore [options] <file>
 @include 'general_options_no_namespace.mdx'
 
 [outage recovery]: /nomad/tutorials/manage-clusters/outage-recovery
-[restore the keyring]: /nomad/docs/operations/key-management#restoring-the-keyring-from-backup
+[restore the keyring]: /nomad/docs/operations/key-management
```

Comment:
Oh, this isn't required as of 1.9.0 and we missed it. We can remove the whole paragraph that links to this.